extensions/spotify-player/package.json

```diff
@@ -406,6 +406,13 @@
           "required": true
         }
       ]
+    },
+    {
+      "name": "startDJ",
```

Comment:
Not sure if this should be disabled by default, as there are a ton of commands already. OTOH, this is a pretty commonly requested feature #12719 #12754 #11437

Reply:
I'm not sure about that either, what do you think would make sense @thomaslombart

Reply:
The DJ feature is already unavailable in some countries (e.g., France), so I would disable it by default.

Reply:
Agree, let's disable it @erics118